openapi/tests/openapi_integration/test_group.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from .helpers.helpers import request_with_validation
+from .helpers.collection_setup import basic_collection_setup, drop_collection
+
+collection_name = 'test_collection_reco'
+
+def upsert_chunked_docs(collection_name, docs=50, chunks=5):
+    points = []
+    for doc in range(docs):
+        for chunk in range(chunks):
+            doc_id = f"doc_{doc}"
+            i = doc * chunks + chunk
+            p = {"id": i, "vector": [0.5, 0.5, 0.5, 0.5], "payload": {"docId": doc_id}} 
+            points.append(p)
+            
+    response = request_with_validation(
+        api='/collections/{collection_name}/points',
+        method="PUT",
+        path_params={'collection_name': collection_name},
+        query_params={'wait': 'true'},
+        body={ "points": points }
+    )
+    
+    assert response.ok
+    
+
+@pytest.fixture(autouse=True, scope="module")
+def setup():
+    basic_collection_setup(collection_name=collection_name)
+    upsert_chunked_docs(collection_name=collection_name)
+    yield
+    drop_collection(collection_name=collection_name)
+
+
+def test_search():
+    response = request_with_validation(
+        api='/collections/{collection_name}/points/search/groups',
+        method="POST",
+        path_params={'collection_name': collection_name},
+        body={
+            "search": {
+                "vector": [1.0, 0.0, 0.0, 0.0],
+                "limit": 10,
+                "with_payload": True,
+            },
+            "group_by": "docId",
+            "top": 3,
```

Comment:
I have doubt about the naming here, `top` vs `limit`.

We have been aliasing `top` to `limit` in the search related APIs.
Because it  works well with a potential `offset` parameters.

I wonder if we should just use `limit` in the group API.

It also clarify the relation between inner limit and outer limit :thinking: